juanmifer72-5949/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Mi primer HTML</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h1> Hello, I'm Juan Miguel Fernández</h1>
+        <h2> I'm self taught and I'm always learning</h2>
+
+        <p>For now JavaScript, html and css are my studies.</p>
+
+        <h3>Table for Linus</h3>
+        <div class="container-titulos">
+            <div class="titulos">Command</div> <div class="titulos">Action</div>
+        </div>
+        <div class="propiedades">
+            <ol>
+                <li>ls</li>
+                <li>whonami</li>
+                <li>date</li>
+                <li>cd</li>
+                <li>pwd</li>
+            </ol>
+        </div>
+        <div class="propiedades">
+            <ul>
+                <li>Show directory</li>
+                <li>Show user</li>
+                <li>Show date</li>
+                <li>Change to directory</li>
+                <li>route directory</li>
+            </ul>
+        </div>
+        
+        <h3>Table of Alias</h3> 
+        <div class="container-titulos">
```

Comment:
I see something that i would change, is that your wedsite language is english but your class names are in spanish,
it could be everything in english imo.
everything else is really neat.